src/main/java/com/celements/cells/CellsClasses.java

```diff
@@ -122,8 +122,9 @@ public DocumentReference getPageLayoutPropertiesClassRef(String wikiName) {
   BaseClass getPageLayoutPropertiesClass() throws XWikiException {
     XWikiDocument doc;
     boolean needsUpdate = false;
-    DocumentReference pageLayoutPropertiesClassRef = cellsClassConfig.getPageLayoutPropertiesClassRef(
-        getContext().getDatabase());
+    DocumentReference pageLayoutPropertiesClassRef = cellsClassConfig
```

Comment:
I do not see how this should be more readable in three lines. please keep it on two.